addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2125,218 +2125,214 @@ we-select.o_we_border_preview_aligned_select {
 
-                &.nw:after,
-                &.se:after {
-                    content: "\f065";
-                    transform: rotate(180deg);
-                }
+                    &.s {
+                        border-top: dashed $o-we-handle-border-width * 0.5 rgba($o-we-handles-accent-color, 0.5);
+                    }
 
-                &.o_handle_start {
+                    .o_handlers_idle &:hover {
+                        border: 0;
+                        background: repeating-linear-gradient(
+                                        45deg,
+                                        rgba($o-we-handles-accent-color, .1),
+                                        rgba($o-we-handles-accent-color, .1) 5px,
+                                        darken(rgba($o-we-handles-accent-color, .25), 5%) 5px,
+                                        darken(rgba($o-we-handles-accent-color, .25), 5%) 10px
+                                    );
+                    }
 
-                    &.w:after,
-                    &.e:after {
-                        content: '\f061';
+                    &:after {
+                        position: absolute;
+                        height: $o-we-handles-btn-size;
+                        content: '';
+                    }
+
+                    &.n:after {
+                        inset: 0 0 auto 0;
+                        transform: translateY(-50%);
                     }
 
-                    &.n:after,
                     &.s:after {
-                        content: '\f063';
+                        inset: auto 0 0 0;
+                        transform: translateY(50%);
                     }
                 }
 
-                &.o_handle_end {
-
-                    &.w:after,
-                    &.e:after {
-                        content: '\f060';
+                &.o_side {
+                    &:before {
+                        content: '';
+                        position: absolute;
+                        inset: 0;
+                        background: $o-we-handles-accent-color;
                     }
 
-                    &.n:after,
-                    &.s:after {
-                        content: '\f062';
+                    &.o_side_x {
+                        &:before {
+                            width: $o-we-handle-border-width;
+                            margin: 0 auto;
+                        }
+
+                        .o_handlers_idle &:hover, &:active {
+                            &:before {
+                                width: $o-we-handle-border-width * 2;
+                            }
+
+                            .o_handle_indicator {
+                                height: $o-we-handles-btn-size * 2;
+                            }
+                        }
                     }
-                }
 
-                &.readonly {
-                    cursor: auto !important;
+                    &.o_side_y {
+                        &:before {
```

Comment:
To fix everywhere: `::before` and `::after` (`::`, not `:`). Even though `:` should work on all browsers, this is not technically correct. I know there were mistakes before but time to make cleaner code :muscle: